components/LoginModal.vue

```diff
@@ -288,4 +287,32 @@ export default {
   display: flex;
   justify-content: center;
 }
+::v-deep .sf-icon-path {
+  margin: 0;
+}
+.login-button {
```

Comment:
font, height, padding should be handled through global button style

Reply:
done